src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
```

Comment:
It's better to use your custom validator.

Reply:
fixed!